FauxPasConfig/main.fauxpas.json

```diff
@@ -1,7 +1,7 @@
 // Generated from the Faux Pas GUI
 {
     // Xcode target to check (String)
-    "target": "StripeiOSStatic",
+    "target": "StripeiOS",
```

Comment:
Did you change this target intentionally? (from what i remember, you have to actually test both targets as they catch slightly different things? I think our ci script does that)